### PR TITLE
nic_hotplug: add a case for hot plug nic with netdev already in use

### PR DESCRIPTION
**Pull request by xiywang**
Signed-off-by: Xiyue Wang xiywang@redhat.com

ID: 1390116

### CODE REVIEW

**Comment by yanan-fu** [comment]
7 commit ?  seems some commit are not related with this case

**Comment by xutian** [comment]
dirty branch, please rebase and update your branch

File: qemu/tests/nic_hotplug.py [hunk 1]
```diff
@@ -90,6 +91,14 @@ def ping_hotplug_nic(ip, mac, session, is_linux_guest):
             status, output = session.cmd_output_safe(del_route_cmd)
         return status, output
 
+    def device_add_nic(pci_model="virtio", netdev="hostnet1", device_id="net1"):
```
Comment (vivianQizhu) [inline qemu/tests/nic_hotplug.py]: doc string

File: qemu/tests/drive_mirror_stress.py [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 import logging
 
-from autotest.client.shared import error
-from autotest.client import utils
-
 from virttest import utils_misc
+from virttest import utils_test
+from virttest import error_context
+
+from avocado.core import exceptions
```
Comment (vivianQizhu) [inline qemu/tests/drive_mirror_stress.py]: avocado before virttest

**Pull request closed without merging**